src/DNSC/Iterative.hs

```diff
@@ -0,0 +1,346 @@
+module DNSC.Iterative where
```

Comment:
You should explicitly export functions here.

Reply:
https://github.com/khibino/dns-resolver/pull/1/commits/579dc30c4899e9c99ff23757560b6d36e8f0444e